### PR TITLE
Index Filter (Search) Release note

### PR DESCRIPTION
**Pull request by JV0812**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [ ] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: blog-service/2023-12-27-index-filter.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - release notes
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We're excited to introduce our new **Index** name filter feature in the search table message footer, which allows you to modify the search query by clicking the index name or view surrounding messages by clicking on the dropdown.<br/><img src={useBaseUrl('img/search/get-started-search/search-page/index-filter.png')} alt="index-filter" width="650" style={{border: '1px solid black'}}/>
```
Comment (ssharma-sumo) [inline blog-service/2023-12-27-index-filter.md:15]: The index field has existed in the system; we have recently started displaying it in the UI along with other metadata values
```
Now, we have included the index field as metadata at the bottom of every message row, along with other metadata. This allows you to modify the search query by clicking the index name or view surrounding messages with the index name by clicking on the dropdown.
```
feel free to updated the text as per your need.

Comment (JV0812) [reply blog-service/2023-12-27-index-filter.md:15]: Thanks for this info @ssharma-sumo. I will get this updated.

File: blog-service/2023-12-27-index-filter.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Index Filed (Search)
```
Comment (sukhhanda1) [inline blog-service/2023-12-27-index-filter.md:2]: I am not sure but just a potential typo here?
```suggestion
title: Index Field (Search)
```

**Review by jpipkin1 — APPROVED** [review]
The release note looks good, @JV0812. 

But I think the feature should also be added to the "[Built-in Metadata](url)" article, and that the release note should link to that article.

File: blog-service/2023-12-29-index-filter.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - release notes
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We're excited to include the **Index** field as metadata at the bottom of every message row, along with other metadata. This allows you to modify the search query by clicking the index name or view surrounding messages by clicking on the dropdown. [learn more](/docs/search/get-started-with-search/search-basics/built-in-metadata)<br/><img src={useBaseUrl('img/search/get-started-search/search-page/index-filter.png')} alt="index-filter" width="650" style={{border: '1px solid black'}}/>
```
Comment (jpipkin1) [inline blog-service/2023-12-29-index-filter.md:15]:
```suggestion
We're excited to include the **Index** field as metadata at the bottom of every message row, along with other metadata. This allows you to modify the search query by clicking the index name or view surrounding messages by clicking on the dropdown. [Learn more].(/docs/search/get-started-with-search/search-basics/built-in-metadata)<br/><img src={useBaseUrl('img/search/get-started-search/search-page/index-filter.png')} alt="index-filter" width="650" style={{border: '1px solid black'}}/>
```

**Comment by JV0812** [comment]
Thanks for your suggestion @jpipkin1, I have also updated the doc associated with this change.

**Pull request merged**